src/widgets/avformatproducerwidget.cpp

```diff
@@ -782,10 +782,28 @@ void AvformatProducerWidget::convert(TranscodeDialog& dialog)
         else
             args << "-map" << "0:a?" << "-map" << "0:V?";
         args << "-map_metadata" << "0" << "-ignore_unknown";
+
+        // Set video filters
+        args << "-vf";
+        QString filterString;
+        QString range;
+        if (ui->rangeComboBox->currentIndex())
+            range = "full";
+        else
+            range = "mpeg";
+        filterString = QString("scale=flags=accurate_rnd+full_chroma_inp+full_chroma_int:in_range=full:out_range=%1").arg(range);
```

Comment:
Why does this change assume the in_range is full? It should use the same as out_range.

Reply:
Typo. Fixed.